kythe-browse.sh

```diff
@@ -15,17 +15,20 @@
 
 usage() {
   cat <<EOF
-$0 verilog file...
+$0 PATH DIR INCLUDE_DIRs
 
-Extracts Kythe facts from the given verilog file and run Kythe web ui to visualize code navigation.
+PATH is the path to the file list.
+DIR is the file list root.
+INCLUDE_DIRs the directories for includes.
+
+Extracts Kythe facts from the given verilog file list and run Kythe web ui to visualize code navigation.
 EOF
 }
 
 set -o pipefail
 BROWSE_PORT="${BROWSE_PORT:-8080}"
 KYTHE_BINDIR="/opt/kythe/tools"
```

Comment:
Somewhere after here:

```shell
[[ -x "$KYTHE_BINDIR"/entrystream ]] || { echo "Missing kythe tool: ..." ; exit 1; }
```

Probably a good idea for every custom installed prerequisite tool that is used.